third_party/accessibility/ax/platform/ax_platform_node_mac.mm

```diff
@@ -767,7 +767,7 @@ - (NSString*)accessibilityLabel {
   // and accessibilityTitle is "the title of the accessibility element"; at
   // least in Chromium, the title usually is a short description of the element,
   // so it also functions as a label.
-  return [self AXTitleInternal];
```

Comment:
Is there a reason to mute the label over title?

Reply:
According to the comment, I shouldn't do it :)

Reply:
> // least in Chromium, the title usually is a short description of the element,
  // so it also functions as a label.

Reply:
probably add a doc mentions why this is an empty string